README.md

```diff
@@ -0,0 +1,56 @@
+# eslint-config-feedzai
+
+This package provides the ESLint configurations as an extensible shared configuration that could be used by everyone and across the Feedzai's products.
+
+This repo includes two distinct ESLint configurations:
+
+- `eslint-config-feedzai`- This package contains our ESLint rules, including the ones for ECMAScript 6+. 
+If you're using React in your projects please check the `eslint-config-feedzai-react`.
```

Comment:
Since the react config is two lines bellow this hardly seems necessary

Reply:
ok, I will remove that